File: LocationAPI/GetLocations.py
import requests
import json

token = "ADD YOUR TOKEN HERE"
page = 1
itemsPerPage = 100
customId = "my-custom-location-id"
type = "construction"

url = ("https://location-api.hibirdi.com/users/{}/locations?page={}&itemsPerPage={}&customId={}&type={}"
       .format(token, page, itemsPerPage, customId, type))

response = requests.request("GET", url)

print(json.dumps(response.json(), indent=2))

File: LocationAPI/GetLocations.js
let data = null;
const xhr = new XMLHttpRequest();

const token = "ADD YOUR TOKEN HERE";
const page = 1;
const itemsPerPage = 100;
const customId = "my-custom-location-id";
const type = "construction";
const url = "https://location-api.hibirdi.com/users/" + token +
            "/locations" +
            "?page=" + page +
            "&itemsPerPage=" + itemsPerPage +
            "&customId=" + customId +
            "&type=" + type;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === 4) {
    console.log(this.responseText);
  }
});

xhr.open("GET", url);
xhr.send(data);
